community/blog/2020.10-Introducing-message-handling-with-Azure-Service-Bus-message-influence/README.md

```diff
@@ -0,0 +1,106 @@
+# Introducing message handling with Azure Service Bus message influence in v0.5
+
+Starting from v0.4 and updated in v0.5, the [Arcus.Messaging](https://github.com/arcus-azure/arcus.messaging) has got an upgrade in handling messages from Azure Service Bus.
+Previous versions already had set up a message handling system where messages could be routed and processed according to contextual information, but the system wasn't yet mature enough that it provided a way to report back to Azure when the message processing was completed.
+
+Until now.
+
+## Azure Service Bus message operations
```

Comment:
I think you're 'deep diving' in this paragraph already, but I -as the reader who has no previous knowledge on Arcus messaging' am already lost.  Might be helpful to have some short high level intro on what Arcus Messaging is or tries to achieve.  (Or, if it is not possible to do that in a few sentences, link to a previous blogpost where Arcus Messaging is introduced ? )

Reply:
Yes, thanks. I added a paragragh with a link to our announcement post.

Reply:
That's a good start indeed @stijnmoreels but we'll need to improve our docs a bit more then I think.
```suggestion
## Providing capability to directly decide what to do with Azure Service Bus messages
```